gcp/modules/gcp-stackdriver-alerting/resources/uptime_checks/preferences.json

```diff
@@ -9,7 +9,8 @@
   },
   "http_check": {
     "path": "/preferences/carla",
-    "port": 80,
+    "use_ssl": ${env == "dev" ? "false" : "true"},
```

Comment:
I think this configuration will be easy to overlook in this location. What do you think of instead using a dedicated variable e.g. `uptime_check_uses_ssl` and setting it as appropriate from `live/[dev|stg|prd]/k8s/stackdriver/alerting`?

Reply:
Sounds good. Your suggestion is a proper way to deal with this kind of interpolations in templates, I just got lazy to do extra piping :)